<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_pot_odds">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mise</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="miseSize">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pot</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="potSize">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_odds_results">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>5</number>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pot Odds (%) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="result">
        <property name="text">
         <string>Résultat</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>5</number>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pot Odds (x:1) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="result_2">
        <property name="text">
         <string>Résultat</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_own_hand">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_own">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_own_hand">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_own1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_own1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_own1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_own2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_own2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_own2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_common_cards">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_common">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>131</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_common1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="card_button_common1">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="card_label_common1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_common2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="card_button_common2">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="card_label_common2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_common3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="card_button_common3">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="card_label_common3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_common4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="card_button_common4">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="card_label_common4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_common5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="card_button_common5">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="card_label_common5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_adv_hand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_adv_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_adv_hand_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_adv1_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv1_1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv1_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_adv1_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv1_2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv1_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin_adv_1">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_adv2_hand">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_adv_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_adv_hand_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_adv2_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv2_1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv2_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_adv2_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv2_2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv2_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin_adv_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_adv3_hand">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_adv_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_adv_hand_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_adv3_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv3_1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv3_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_adv3_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv3_2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv3_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin_adv_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_adv4_hand">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_adv_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_adv_hand_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_adv4_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv4_1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv4_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_adv4_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv4_2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv4_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin_adv_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_adv5_hand">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>380</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QSplitter" name="splitter_adv_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_adv_hand_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_adv5_1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv5_1">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv5_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_adv4_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPushButton" name="card_button_adv5_2">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/back.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="card_label_adv5_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_combin_adv_5">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
